### DISCUSSION TITLE
Mongoose 7.9 release has a missing function

### DISCUSSION THREAD

**Discussion by hdabis1234** [cesanta/mongoose#2087] [Ask questions, get answers] [answered]
Can somebody tell me if mg_straddr() has been removed from mongoose.c in 7.9.

**Comment by scaprile** [comment, marked as answer]
It is not a missing function: it is not in the code, it is not in the headers, it is not in the docs; it has been removed by #1897 
Please be aware that %I has been removed in latest

**Reply by tomboness** [reply]
Thanks for the heads up about the removal of %I.  mg_straddr() was used in the examples in 7.8 and it was mentioned in the release notes of 7.9 and so caused me some confusion when migrating to 7.9.

**Reply by scaprile** [reply]
Oops, sorry about that... yes, we fixed a stack overflow on that function and later we removed it
